packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -99,7 +102,8 @@ class CreateCommand extends FlutterCommand {
     );
     printStatus('Wrote $generatedCount files.');
 
-    printStatus('');
+    updateXcodeGeneratedProperties(dirPath, BuildMode.debug, flx.defaultMainPath);
+    printStatus('Wrote Xcode settings.');
```

Comment:
Probably wouldn't print this. The output on flutter create is somewhat sparse. This might stand out and confuse the user. There's nothing really actionable for the user.